diff --git a/vision_apps/platform/j722s/rtos/app_mem_map.h b/vision_apps/platform/j722s/rtos/app_mem_map.h
index f08ef0f..d694dad 100755
--- a/vision_apps/platform/j722s/rtos/app_mem_map.h
+++ b/vision_apps/platform/j722s/rtos/app_mem_map.h
@@ -264,7 +264,7 @@
 #define DDR_C7X_2_SCRATCH_PHYS_SIZE (0x04000000u)
 
 /* memory for shared memory buffers in high DDR [ size 512.00 MB ] */
-#define DDR_SHARED_MEM_PHYS_ADDR (0x900000000u)
+#define DDR_SHARED_MEM_PHYS_ADDR (0xC0000000u)
 #define DDR_SHARED_MEM_PHYS_SIZE (0x20000000u)
 
 #define DDR_64BIT_BASE_VADDR (0x100000000u)
