diff --git a/vision_apps/platform/j722s/rtos/c7x_1/freertos.syscfg b/vision_apps/platform/j722s/rtos/c7x_1/freertos.syscfg
index 752338e..639c1bf 100644
--- a/vision_apps/platform/j722s/rtos/c7x_1/freertos.syscfg
+++ b/vision_apps/platform/j722s/rtos/c7x_1/freertos.syscfg
@@ -168,7 +168,7 @@ mmu_armv89.attribute         = "MAIR7";
  */
 mmu_armv810.$name             = "DDR_SHARED_MEM";
 mmu_armv810.vAddr             = 0xC0000000;
-mmu_armv810.pAddr             = 0x900000000;
+mmu_armv810.pAddr             = 0xC0000000;
 mmu_armv810.size              = 0x20000000;
 mmu_armv810.attribute         = "MAIR7";
 
diff --git a/vision_apps/platform/j722s/rtos/c7x_1/freertos_no_board_deps.syscfg b/vision_apps/platform/j722s/rtos/c7x_1/freertos_no_board_deps.syscfg
new file mode 120000
index 0000000..823c460
--- /dev/null
+++ b/vision_apps/platform/j722s/rtos/c7x_1/freertos_no_board_deps.syscfg
@@ -0,0 +1 @@
+freertos.syscfg
\ No newline at end of file
diff --git a/vision_apps/platform/j722s/rtos/c7x_1/generated/ti_dpl_config.c b/vision_apps/platform/j722s/rtos/c7x_1/generated/ti_dpl_config.c
index 46567b6..7dd844a 100644
--- a/vision_apps/platform/j722s/rtos/c7x_1/generated/ti_dpl_config.c
+++ b/vision_apps/platform/j722s/rtos/c7x_1/generated/ti_dpl_config.c
@@ -200,7 +200,7 @@ MmuP_RegionConfig gMmuRegionConfig[CONFIG_MMU_NUM_REGIONS] =
     },
     {
         .vaddr = 0xC0000000u,
-        .paddr = 0x900000000u,
+        .paddr = 0xC0000000u,
         .size = 0x20000000u,
         .attr = {
             .accessPerm = MMUP_ACCESS_PERM_PRIV_RW_USER_NONE,
diff --git a/vision_apps/platform/j722s/rtos/c7x_2/freertos.syscfg b/vision_apps/platform/j722s/rtos/c7x_2/freertos.syscfg
index 3a91b95..7748aeb 100644
--- a/vision_apps/platform/j722s/rtos/c7x_2/freertos.syscfg
+++ b/vision_apps/platform/j722s/rtos/c7x_2/freertos.syscfg
@@ -168,7 +168,7 @@ mmu_armv89.attribute         = "MAIR7";
  */
 mmu_armv810.$name             = "DDR_SHARED_MEM";
 mmu_armv810.vAddr             = 0xC0000000;
-mmu_armv810.pAddr             = 0x900000000;
+mmu_armv810.pAddr             = 0xC0000000;
 mmu_armv810.size              = 0x20000000;
 mmu_armv810.attribute         = "MAIR7";
 
diff --git a/vision_apps/platform/j722s/rtos/c7x_2/freertos_no_board_deps.syscfg b/vision_apps/platform/j722s/rtos/c7x_2/freertos_no_board_deps.syscfg
new file mode 120000
index 0000000..823c460
--- /dev/null
+++ b/vision_apps/platform/j722s/rtos/c7x_2/freertos_no_board_deps.syscfg
@@ -0,0 +1 @@
+freertos.syscfg
\ No newline at end of file
diff --git a/vision_apps/platform/j722s/rtos/c7x_2/generated/ti_dpl_config.c b/vision_apps/platform/j722s/rtos/c7x_2/generated/ti_dpl_config.c
index 4455ddf..db18761 100644
--- a/vision_apps/platform/j722s/rtos/c7x_2/generated/ti_dpl_config.c
+++ b/vision_apps/platform/j722s/rtos/c7x_2/generated/ti_dpl_config.c
@@ -200,7 +200,7 @@ MmuP_RegionConfig gMmuRegionConfig[CONFIG_MMU_NUM_REGIONS] =
     },
     {
         .vaddr = 0xC0000000u,
-        .paddr = 0x900000000u,
+        .paddr = 0xC0000000u,
         .size = 0x20000000u,
         .attr = {
             .accessPerm = MMUP_ACCESS_PERM_PRIV_RW_USER_NONE,
